### ISSUE TITLE
Should the debugger reset world.tick_usage on resume?

### ISSUE THREAD

**Issue by DamianX** [MCHSL/extools#42]
I'm trying to debug an issue on /vg/station and I find myself pausing the world in the middle of an a subsystem's Initialize().
When I'm done I hit "resume" and the game runs until `CHECK_TICK` runs, which notices the current `world.tick_usage` is several orders of magnitude higher than what's normal, and therefore `sleep()`s for an excessively long time to compensate. I inserted a debug statement inside `stoplag()` to prove this:
```
## TESTING: i: 1, tick_lag: 1, delta_calc: 2.29, cpu: 158, tick_usage: 229
## TESTING: i: 1, tick_lag: 1, delta_calc: 1.19, cpu: 119, tick_usage: 99
## TESTING: i: 1, tick_lag: 1, delta_calc: 1, cpu: 92, tick_usage: 99
## TESTING: i: 1, tick_lag: 1, delta_calc: 1360.29, cpu: 75, tick_usage: 136029
```

TL;DR:
Time spent looking at the debugger increases `world.tick_usage`, perhaps it would be better if it didn't.
I don't know if that's feasible at all.

**Comment by MCHSL** [comment]
I think I'd need to hook the function that's responsible for fetching tick_usage, then have it return faked values. GEtting these fake values right be a problem though (or we could just return 50% always or whatever)

**Comment by SpaceManiac** [comment]
I ran into the `stoplag` problem myself. We should probably try to address this.

**Comment by AffectedArc07** [comment]
I thought this was semi related given how the MC was dysfunctional after I had paused for a long time. Glad I know this is a confirmed issue.
